lib/stat_tracker.rb

```diff
@@ -280,6 +320,18 @@ def most_accurate_team(season_id)
     find_team(best).name
   end
 
+  def winningest_coach(season_id)
+    find_all_wins_by_coach(season_id).max_by do |coach, percent_wins|
+      percent_wins
+    end[0]
+  end
```

Comment:
I think you should be using the `percent_wins_by_coach` here instead of `find_all_wins_by_coach` right?